lib/config.js

```diff
@@ -482,6 +487,11 @@ Config.prototype.update = function (options) {
     this.skip_signing = true
   }
 
+  if (options.build_delta_installer) {
+    this.build_delta_installer = true
+    this.last_chrome_installer = options.last_chrome_installer
```

Comment:
I think we should be checking this for a non-empty value instead of just assuming it exists. Presumably it should be an error if build_delta_installer == true && last_chrome_installer === null ?

Reply:
or is there a default value we should be providing here? https://github.com/brave/brave-browser/pull/9667/files#diff-1bb8ffaf71730e8067757db1a6a153dbR85

Reply:
Right, `last_chrome_installer` should have args for building delta installer.
Added `assert` here - https://github.com/brave/brave-browser/pull/9667/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R197.
It doesn't make sense to try to build delta installer w/o `last_chrome_installer` args.